packages/upload-core/src/upload.js

```diff
@@ -215,7 +215,7 @@ class Upload {
       this.onProgress.forEach(cb => cb());
       this.timeoutId = setTimeout(() => {
         this.scan();
-      }, 1000);
+      }, this.options.pollingTime || 5000);
```

Comment:
Let's add the default `pollingTime` to the `defaultOptions` object that is already being leveraged in this class.